modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -147,8 +145,10 @@ public ObjectEntry addObjectEntry(
 
 		validateReadOnlyObjectFields(null, objectDefinition, objectEntry);
 
-		ServiceContext serviceContext = _createServiceContext(
-			dtoConverterContext, objectDefinition, objectEntry);
+		ServiceContext serviceContext = ServiceContextUtil.createServiceContext(
+			dtoConverterContext, objectDefinition, objectEntry, null,
+			_resourceActionLocalService, _resourcePermissionLocalService,
+			_roleLocalService);
```

Comment:
Instead of passing null, why don't you pass `objectEntry.getPermissions()` ?
I prefer to do this rather than doing the GET in the ServiceContextUtil.createServiceContext() method.